sp/src/game/server/ez2/npc_basepredator.cpp

```diff
@@ -1262,7 +1275,7 @@ NPC_STATE CNPC_BasePredator::SelectIdealState( void )
 	case NPC_STATE_COMBAT:
 	{
 		// COMBAT goes to ALERT upon death of enemy
-		if ( GetEnemy() != NULL && ( HasCondition( COND_LIGHT_DAMAGE ) || HasCondition( COND_HEAVY_DAMAGE ) ) && IsPrey( GetEnemy() ) )
+		if ( GetEnemy() != NULL && ( HasCondition( COND_LIGHT_DAMAGE ) || HasCondition( COND_HEAVY_DAMAGE ) ) && IsPrey( GetEnemy() ) && gpGlobals->curtime - m_flLastHurtTime < 1.0f )
```

Comment:
Why is this exactly? Could this lead to situations where the Gonome stays in combat state a second longer before returning to alert during chapter 3 or other stealth based sections?

Reply:
I need to read more carefully. I totally skipped over the part where this is for when the current enemy is prey

Reply:
I will ask, why is this within one second? What case was this written for?

Reply:
Could you leave a comment explaining what ``gpGlobals->curtime - m_flLastHurtTime < 1.0f`` is for? I feel like it's counterintuitive